.github/workflows/tests.yml

```diff
@@ -5,6 +5,12 @@ on:
 
 name: Continuous integration
 
+env:
+  # This is required because commands like `cargo check` do not
+  # read the cargo development dependencies.
+  # See https://github.com/Arnavion/k8s-openapi/issues/132
+  K8S_OPENAPI_ENABLED_VERSION: "1.24"
```

Comment:
I don't like this repetion between GHA and makefile, but I can live with it.

The alternative that comes to mind is a bit clunky. Would be to create a new job or step:
```yaml
  jobs:
    read_openapi_var:
      steps:
        - run: echo "openapi_var=$(sed  as needed Makefile)" >> $GITHUB_OUTPUT
```
And then on all steps consume that output as:
```yaml
env:
  K8S_OPENAPI_ENABLED_VERSION: ${{ steps.read-openapi_var.outputs.openapi_var }}
```

Reply:
TBH, the version is not really relevant... Even if they diverge nothing bad is going to happen because the tests do not use any Kubernetes object at all